src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -75,7 +75,7 @@
 
 _AZ_LOGIN_MESSAGE = "Please run 'az login' to setup account."
 
-_USE_VENDORED_SUBSCRIPTION_SDK = True
+_USE_VENDORED_SUBSCRIPTION_SDK = False
```

Comment:
Since the vendor SDK has been deleted, why do we still keep the logic of this switch?

Reply:
In case there is a track 3 someday. 😏

Reply:
😂😂